### DISCUSSION TITLE
Pagination: Translation

### DISCUSSION THREAD

**Discussion by BubeDameKoenig** [livewire/flux#327] [Feature Requests] [answered]
The following words are fixed in `pagination`: `Showing`, `to`, `of`, `entries`
I could publish pagination and changing it there but then I would miss the updates.

Couldn't you change this (`pagination.showing`) and point out in the documentation that you still have to add these four translations individually to the file?

**Comment by joshhanley** [comment, marked as answer]
I've just reviewed the pagination component and it now has the correct translation strings, so I'm going to close this.
